bin/kramdown-rfc-clean-svg-ids

```diff
@@ -18,6 +18,7 @@ def svg_clean_ids(s)
 rescue => detail
   warn "*** Can't clean SVG: #{detail}"
   d.to_s
+  exit 1
```

Comment:
This means there will be no output while the previous line was careful preparing to return something useful.

I'm not even sure that failing in this case warrants an exit code 1.

Reply:
Yeah, it makes sense not to exit with `1`.